doc/nrf/device_guides/working_with_nrf/nrf54l/peripheral_sensor_node_shield.rst

```diff
@@ -0,0 +1,81 @@
+.. _peripheral_sensor_node_shield:
+
+Developing with Peripheral Sensor node shield
+#############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Peripheral Sensor node shield is an extension for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>`.
+It is a testing shield with following sensors:
```

Comment:
```suggestion
It is a testing shield designed for evaluating I2C and SPI protocol drivers. 
It includes the following sensors:
```

I just noticed that (if I'm not mistaken) the line 16 seems to be a general remark. So it's probably better to move it here